drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -2265,27 +2268,32 @@ void RasterizerStorageGLES2::mesh_add_surface(RID p_mesh, uint32_t p_format, VS:
 				}
 
 				if (p_format & VS::ARRAY_COMPRESS_VERTEX) {
-					attribs[i].type = _GL_HALF_FLOAT_OES;
-					stride += attribs[i].size * 2;
-					uses_half_float = true;
+					attribs[i].type = GL_HALF_FLOAT;
```

Comment:
I'm guessing this is a copy-paste error from GLES3. This line is causing the CI to fail because it should be ``_GL_HALF_FLOAT_OES``